README.md

```diff
@@ -129,6 +137,7 @@ For each fo the "deploy" scripts, there is an equivalent "cleanup" script you ca
 If you want to clean the entire setup, first source your `env.sh` script, and then run the scripts in the following order
 
 ```bash
+./cleanup-mint.sh # if applicable
```

Comment:
Can we include the same in the docs/cloudshell-tutorial.md as well?

Reply:
good shout. Done.